sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -3778,3 +3778,37 @@ test_suite(
     tags = ["manual"],
     tests = ["rom_ext_upgrade_interrupt_{}".format(lc_state) for lc_state, _ in get_lc_items()],
 )
+
+opentitan_functest(
```

Comment:
If this is solely to test the infrastructure, then let move this to `sw/device/test/BUILD`.